dist/index.js

```diff
@@ -2414,7 +2447,7 @@ exports.Octokit = Octokit;
 
 Object.defineProperty(exports, "__esModule", ({ value: true }));
 
-var isPlainObject = __nccwpck_require__(558);
+var isPlainObject = __nccwpck_require__(3287);
```

Comment:
- [ ] Is that a safe thing to change the port ?

Reply:
Looked like a mail server port or something but maybe it isn't 😄 
All good if you could confirm it works 👍